.lintrunner.toml

```diff
@@ -250,6 +250,7 @@ command = [
     '@{{PATHSFILE}}'
 ]
 exclude_patterns = [
+    'tools/torchfix/**',
```

Comment:
I'm a bit confused. This seems to _exclude_ a folder from the linter, meaning it should've been working everywhere before.

But if that's the case, why wasn't it actually getting linted before? Or do we not run this linter as part of CI yet?

Reply:
"we not run this linter as part of CI yet" - that's my understanding for the repo-wide linter.

For TorchFix, there is already a separate workflow to run relevant linters in the specific configuration.